src/style.css

```diff
@@ -1 +1,47 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src:
+    url(fonts/Avenir-Book.ttf) format(ttf),
+    url(fonts/Avenir-Heavy.ttf) format(ttf),
+    url(fonts/Avenir.ttc) format(ttc);
+}
+
+.search-bar {
+  display: flex;
+  width: 100%;
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px #3d4e611a;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  border: solid 1px #e1e7ed;
+  border-radius: 3px;
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+  color: #3d4e61;
+}
+
+.search-bar__big {
+  padding: 25px 62px 26px;
+  background-position: 26px 25px;
+  background-size: 19px 19px;
+  margin: 20px 0;
+  font-size: 16px;
+  height: 70px;
```

Comment:
Don't fix container size (if there is no such a requirement). By setting an explicit height, you may encounter issues with content overflow or clipping if the content exceeds the container's height. Consider using padding and let the content dictate the height.